gallery_dl/extractor/instagram.py

```diff
@@ -140,6 +142,65 @@ def _extract_postpage(self, url):
 
         return medias
 
+    def _extract_stories(self, url):
+        page = self.request(url).text
+        shared_data = self._extract_shared_data(page)
+
+        # If no stories are present the URL redirect to `ProfilePage'
+        if 'StoriesPage' not in shared_data['entry_data']:
+            return []
+
+        user_id = shared_data['entry_data']['StoriesPage'][0]['user']['id']
+        variables = '{' + \
+            '"reel_ids":["{}"],'.format(user_id) + \
+            '"tag_names":[],"location_ids":[],' + \
+            '"highlight_reel_ids":[],"precomposed_overlay":true,' + \
+            '"show_story_viewer_list":true,' + \
+            '"story_viewer_fetch_count":50,"story_viewer_cursor":"",' + \
+            '"stories_video_dash_manifest":false}'
```

Comment:
You can write multi-line strings in Python by putting all string "fragments" in brackets.
```
variables = (
    "foo"
    "bar"
    "baz"
)
assert variables == "foobarbaz"
```

Reply:
Thanks, I have adjusted it as suggested!